recipes-kernel/linux/linux-hikey_4.5.bb

```diff
@@ -0,0 +1,18 @@
+require linux-hikey.inc
+
+DESCRIPTION = "Hisilicon 4.5 Kernel"
+
+PV = "4.5+git${SRCPV}"
+
+SRCREV_kernel="623d1d7bbd697e5793a135d5334e02c0e9889ba1"
+
+SRC_URI = "git://github.com/xin3liang/linux;branch=hikey-tracking-integration-devel-drm;name=kernel \
+           file://defconfig  \
```

Comment:
We probably don't want to use this tree but the mainline-rebase branch. This topic branch is meant to land there.